rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -960,6 +937,43 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		propagate_used_devices();
 	};
 
+	const auto get_audio_output_devices = [this]()
+	{
+		auto dev_enum = Emu.GetCallbacks().get_audio_enumerator(static_cast<audio_renderer>(ui->audioOutBox->currentIndex()));
```

Comment:
Use currentData().toInt()
The data is not necessarily in the same order ahs the GUI index.

Reply:
This always returns 0 now

Reply:
sorry. it's probably:
```suggestion
		const QVariantList var_list = ui->audioOutBox->currentData().toList();
		ensure(var_list.size() == 2 && var_list[1].canConvert<int>());
		auto dev_enum = Emu.GetCallbacks().get_audio_enumerator(var_list[1].toInt());
```